### DISCUSSION TITLE
[功能请求]: 大字倒计时（考试用）

### DISCUSSION THREAD

**Discussion by kaokao221** [ClassIsland/ClassIsland#555] [插件功能请求]
### 检查清单

- [X] 我已经在 [Issues](https://github.com/HelloWRC/ClassIsland/issues?q=) 中找过我要提出的问题，没有找到相同问题的ISSUE。
- [X] 我已经详细阅读过应用的帮助文档，但我想要的功能没有实现。
- [x] 我已确认我的功能尚未在 [dev](https://github.com/HelloWRC/ClassIsland/commits/dev/) 分支实现。

### 背景与动机

我们学校有一个奇怪的东西叫做作业考试化

### 想要实现或优化的功能

针对作业考试化，希望可以在ClassIsland上显示（最好是比较醒目的）大字精确到秒的倒计时

**Comment by HelloWRC** [comment]
感谢您的建议，不过这个功能可能有点超出ClassIsland的开发目标了。未来可能会通过扩展的方式实现这个功能。

**Comment by unDefFtr** [comment]
> 感谢您的建议，不过这个功能可能有点超出ClassIsland的开发目标了。未来可能会通过扩展的方式实现这个功能。

这还插件？（
和考试看板联动得了（

**Comment by jiangyin14** [comment]
这还插件？（
和考试看板联动得了（

**Comment by unDefFtr** [comment]
@hello8693DSZ （
找微软草给你开个socket或者咋滴（



---- Replied Message ----
| From | ***@***.***> |
| Date | 10/08/2024 23:02 |
| To | ***@***.***> |
| Cc | ***@***.***>***@***.***> |
| Subject | Re: [ClassIsland/ClassIsland] [功能请求]: 大字倒计时（考试用） (Issue #68) |

这还插件？（
和考试看板联动得了（

**Comment by DryIce-cc** [comment]
> @undefftr 找微软草给你开个socket或者咋滴（

wrc 已经整好 [IPC 事件广播](https://github.com/ClassIsland/ClassIsland/commits/master/ClassIsland.Shared.IPC) 了（

**Comment by unDefFtr** [comment]
事件广播没用的说，得双向控制（



---- Replied Message ----
| From | ***@***.***> |
| Date | 10/09/2024 19:16 |
| To | ***@***.***> |
| Cc | ***@***.***>***@***.***> |
| Subject | Re: [ClassIsland/ClassIsland] [功能请求]: 大字倒计时（考试用） (Issue #68) |

@undefftr 找微软草给你开个socket或者咋滴（

wrc 已经整好 IPC 事件广播 了（

**Comment by Makitoid** [comment]
你去试试https://github.com/ProjectCampus-CH/exam-showboard-next

**Comment by HUSX100** [comment]
可以用自动化+Rainmeter 动态桌面插件（当科目为考试时，启动 Rainmeter）
